src/components/CoordinatorHome.js

```diff
@@ -124,40 +120,67 @@ const CoordinatorHome = observer(({ assembly }) => (
               ) }
             </Popover.Container>
           </div>
+          
+          {/* TODO: Last Reported Day */}
+          {/* <Cell>
+            { participant
+                .medication_reports
+                .sort(function(a,b){
+                  return DateTime.fromISO(b.created_at) - DateTime.fromISO(a.created_at);
+                }).toString()
+            }
+          </Cell> */}
 
-          <NameLinkCell onClick={() => assembly.participant_history.watch(participant.uuid, () => assembly.currentPage = CoordinatorParticipantHistory)}>
+          <NameLinkCell onClick={() =>
```

Comment:
```suggestion
          <NameLinkCell
            onClick={() =>
```

...and then fix up the spacing on the next two lines. Otherwise, :+1:.